packages/react-ui-components/src/components/SchemaForm/Widgets/CheckboxesWidget/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface ICheckboxConfig {
+	multiple: boolean;
+	allButton: boolean;
+	minItems: number;
+	maxItems: number;
+}
```

Comment:
you can probably add the `readonly` and `disabled` props so you can use them inside helpers.